README.md

```diff
@@ -19,6 +17,62 @@ features:
 For more details, please see the
 [documentation](https://kubernetes-csi.github.io/docs/Setup.html#csidriver-custom-resource-alpha).
 
+## Compatibility
+
+| Latest stable release                                                                                       | Branch                                                                                  | Compatible with CSI Version                                                                | Container Image                                 | Min K8s Version | Max K8s Version |
+| ----------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------ | ----------------------------------------------- | --------------- | --------------- |
+| [cluster-driver-registrar v1.0.1](https://github.com/kubernetes-csi/cluster-driver-registrar/releases/tag/v1.0.1) | [release-1.0](https://github.com/kubernetes-csi/cluster-driver-registrar/tree/release-1.0) | [CSI Spec v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | quay.io/k8scsi/csi-cluster-driver-registrar:v1.0.1 | 1.13            | -               |
+
+## Usage
+
+* CSI driver UNIX domain socket:
```

Comment:
Maybe we don't need to talk about this here, and instead just mention it in the `--csi-address` section below.  We explicitly called it out for node-driver-registrar because having two sockets was very confusing.